test/hardhat/distributionCreator/distributionCreator.test.ts

```diff
@@ -447,7 +469,7 @@ contract('DistributionCreator', () => {
         propToken1: 2000,
         propFees: 4000,
         isOutOfRangeIncentivized: 0,
-        epochStart: startTime,
+        epochStart: startTime + 1000,
```

Comment:
why did we have to increase this?

Reply:
because else the startTime is in the past compared to the current timestamp